csharp-basic-topics/AsAndIsOperatorsInCSharp/AsAndIsOperatorsInCSharpTests/AsAndIsOperatorsTests.cs

```diff
@@ -0,0 +1,99 @@
+using Application;
+using System;
+using System.IO;
+using Xunit;
+
+namespace AsAndIsOperatorsInCSharpTests
+{
+    public class AsAndIsOperatorsTests : IDisposable
+    {
+        private readonly StringWriter output = new();
+
+        public AsAndIsOperatorsTests()
+        {
+            Console.SetOut(output);
+        }
+
+        public void Dispose()
+        {
+            output.Dispose();
+            GC.SuppressFinalize(this);
+        }
+
+        public string[] PrintedOutputToArray()
+        {
+            var printedString = output.ToString();
+            return printedString.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
+        }
+
+        [Fact]
+        public void EqualityVsTypeComparison_ShouldPrintThatComparisonsAreValid()
```

Comment:
A small change to test names. We have a convention we're trying to use, so check the guidelines, please. I think this is clear enough as it is, but let's try to use the same approach as everyone else for consistency.

Reply:
No problem, already changed it. Thanks for the feedback!